tests/tests_selinux_modules.yml

```diff
@@ -127,7 +125,7 @@
           assert:
             that: test_module_c == 'absent'
       when: ansible_distribution_major_version < "7" and
-        ( ansible_distribution == "CentOS" or ansible_distribution == "RedHat" )
+        ansible_distribution in ["CentOS", "RedHat"]
```

Comment:
This isn't applicable for Rocky?

Reply:
See my other comment about Rocky Linux 8.4.